src/main/scala/treadle/executable/ClockInfo.scala

```diff
@@ -10,6 +10,6 @@ case class ClockInfo(
 
 object ClockInfo {
   val DefaultName: String = "clock"
-  val DefaultPeriod: Long = 1000L
-  val DefaultOffset: Long = 500L
+  val DefaultPeriod: Long = 10L
```

Comment:
Could we have a comment (or better yet, some ScalaDoc) on what the units are, or what reasonable values would be?